lib/internal/child_process.js

```diff
@@ -230,12 +230,12 @@ util.inherits(ChildProcess, EventEmitter);
 
 
 function flushStdio(subprocess) {
-  if (subprocess.stdio == null) return;
-  subprocess.stdio.forEach(function(stream, fd, stdio) {
-    if (!stream || !stream.readable || stream._readableState.readableListening)
-      return;
-    stream.resume();
-  });
+  if (subprocess.stdio === null || subprocess.stdio === undefined) return;
```

Comment:
Motivation of this change was [this line](https://github.com/notarseniy/node/blob/0b3efdc5de0abd93e02cbbfa092963ee97e492cb/lib/internal/child_process.js#L804) with 'that-style' condition.